R/mboostLSS.R

```diff
@@ -204,7 +254,7 @@ mboostLSS_fit <- function(formula, data = list(), families = GaussianLSS(),
                 ## update value of nuisance parameters
                 ## use response(fitted()) as this is much quicker than fitted(, type = response)
                 for( k in mods[-best]) {
-                    assign(names(fit)[best], families[[best]]@response(fitted(fit[[best]])),
+                    assign(names(fit)[best], families[[best]]@response(mboost:::fitted.mboost(fit[[best]])),
```

Comment:
see above. preferably use the old notation (unless you have good reasons; then please explain them :))